### ISSUE TITLE
nk_fill_rect has bleeding edges

### ISSUE THREAD

**Issue by LennertSchneider** [vurtun/nuklear#424]
Hi,

doing e.g.
nk_fill_rect(nk_window_get_canvas(ctx), nk_rect(0,20,2,580), 0.0f, nk_rgb(255,255,255));
will not draw the expected rect on exact pixels using white color without transparency, but instead draws "between the pixels" resulting in half-transparency of the drawn rect. i think the transformation to screen coords is off by 0.5 or something like that.

Cheers.

**Comment by vurtun** [comment]
Do you use anti-aliasing? If so please try turning it off for shapes in `nk_convert_config`:
```c
 `config.shape_AA = NK_ANTI_ALIASING_OFF;`
````

**Comment by LennertSchneider** [comment]
yes, it was turned on. and i'd like to have it turned on, e.g. for circles.
however i tried turning it off which gets rid of the in-between issue, but then the rect is only 1px in height, instead of 2. so something is still off there... :/
